internal/controller/api/management_test.go

```diff
@@ -350,4 +350,45 @@ var _ = Describe("Management", func() {
 
 	})
 
+	Describe("Connecting to the connection list endpoint with account identifier", func() {
+		Context("With a valid identity header", func() {
+			It("Should be able to get a list of open connections for provided account", func() {
+
+				req, err := http.NewRequest("GET", CONNECTION_LIST_ENDPOINT+"/"+CONNECTED_ACCOUNT_NUMBER, nil)
+				Expect(err).NotTo(HaveOccurred())
+
+				req.Header.Add(IDENTITY_HEADER_NAME, validIdentityHeader)
+
+				rr := httptest.NewRecorder()
+
+				ms.router.ServeHTTP(rr, req)
+
+				Expect(rr.Code).To(Equal(http.StatusOK))
+
+				// FIXME: need to verify that connection list
```

Comment:
The "FIXME" isn't needed as the following code verifies the connection list.  Right?

Reply:
Correct. Forgot to take that out